final_prep/annotations.rb

```diff
@@ -18,15 +20,17 @@ def build_a_bear(name, age, fur, clothes, special_power)
   }
   return built_bear
 end
-
+# build_a_bear defining paraments with arguments
```

Comment:
This is not where parameters are defined (they are defined in line 7). You might say "the method is _called_ with arguments"